### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by divyanshkul**
* Added some more info to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
-an IDE for Android app development
-3. Import the cloned repository into your IDE
+You can run the application by two ways:
 
+1. Via your own android smartphone
+     - Enable [USB Debuuging](https://www.howtogeek.com/129728/how-to-access-the-developer-options-menu-and-enable-usb-debugging-on-android-4.2/) in your phone.
+	 - Click **Run** on the Android Studio tool bar, or **Shift + F10** to run the app.
 
+2. By running a virtual device.
+	 - Setup a [Android Virtual Device](https://developer.android.com/studio/run/managing-avds.html) in the IDE.
```
Comment (garg-anuj) [inline README.md]: Please correct indentation here

**Comment by divyanshkul** [comment]
@lakhina @garg-anuj Resolved merge conflicts. :)

File: README.md [hunk 2]
```diff
@@ -1,29 +1,46 @@
...
-[Compressor Head](https://github.com/jboss-outreach/compressor-head)
+Compressor Head is a fast web application based on [Python](https://www.python.org/)to resize images.
+For a more detailed explanation, visit the [Compressor Head](https://github.com/jboss-outreach/compressor-head)
 project.
 
+## Setting up the Project
+1. Fork the repository by clicking on the Fork icon at the top right corner of the repository page.
+2. Clone the repository on to your local machine by running the following commands on git:
+		
+		`git clone git clone https://github.com/[YOUR-USERNAME]/compressor-head-android.git`
```
Comment (lakhina) [inline README.md]: Correct this git command

Comment (divyanshkul) [reply README.md]: @lakhina these are showing up to be correct in mine, any idea why this is happening?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
+3. Refer [Forking and Cloning in git](https://help.github.com/articles/fork-a-repo/) if you are stuck somewhere.
+4. Download and install [Android Studio](https://developer.android.com/studio/index.html) which is an IDE for android application development.
+5. You will also need to download the Android SDK from the IDE itself.
+6. Import the **forked** repository into Android Studio by clicking on files-->Open and navigate to the directory where you forked the repository.
+
+## Running the Application
+You can run the application by two ways:
+
+1. Via your own android smartphone
+     - Enable [USB Debuuging](https://www.howtogeek.com/129728/how-to-access-the-developer-options-menu-and-enable-usb-debugging-on-		android-4.2/) in your phone.
```
Comment (lakhina) [inline README.md]: Correct this link

Comment (divyanshkul) [reply README.md]: @lakhina yes that is an error, can I change it now, after the merge or do i have to submit a new PR?

**Pull request merged**